src/openzaak/components/catalogi/api/filters.py

```diff
@@ -122,7 +122,7 @@ class ZaakTypeFilter(FilterSet):
         field_name="concept", method=status_filter, help_text=STATUS_HELP_TEXT
     )
     trefwoorden = CharArrayFilter(field_name="trefwoorden", lookup_expr="contains")
-    identificatie = filters.NumberFilter(field_name="identificatie")
+    identificatie = filters.CharFilter(field_name="identificatie")
```

Comment:
sorry to keep nagging, but do we even have to explicitly provide this? Isn't listing this just in `fields` sufficient?